client/src/shared/Form/AddTelephone.js

```diff
@@ -13,26 +25,91 @@ const AddTelephone = ({ className, formik }) => {
+const AddTelephone = ({ className, formik }) => {
+  const classes = useStyles();
   return (
     <div>
-      <p style={mystyle}>Ваш номер телефону</p>
-      <MuiPhoneNumber
+      <p className={classes.mystyle}>Ваш номер телефону</p>
+
+      <FieldArray
         name="phone"
-        className={className}
-        value={formik.values.phone}
-        onChange={value =>
-          formik.setFieldValue('phone', value)
-        }
-        onBlur={formik.handleBlur}
-        defaultCountry="ua"
-        regions="europe"
-        helperText={
-          formik.errors.phone &&
-          formik.touched.phone &&
-          formik.errors.phone
-        }
-        error={formik.errors.phone && formik.touched.phone}
+        render={arrayHelpers => (
+          <div className={classes.paddingTop}>
+            {formik.values.phone &&
+            formik.values.phone.length > 0 ? (
+                formik.values.phone.map((phone, index) => {
+                  const errorMassage = getIn(
+                    formik.errors,
+                    `phone[${index}]`
+                  );
+                  const touched = getIn(
+                    formik.touched,
+                    `phone[${index}]`
+                  );
+                  return (
+                    <div key={generate()}>
+                      <MuiPhoneNumber
+                        name={`phone[${index}]`}
+                        className={className}
+                        value={phone}
+                        onChange={value =>
+                          formik.setFieldValue(
+                            `phone[${index}]`,
+                            value
+                          )
+                        }
+                        onBlur={formik.handleBlur}
+                        defaultCountry="ua"
+                        regions="europe"
+                        helperText={
+                          errorMassage &&
+                        touched &&
+                        errorMassage
+                        }
+                        error={errorMassage && touched}
+                        InputProps={{
+                          endAdornment: (
+                            <InputAdornment
+                              position="end"
+                              onClick={() =>
+                                arrayHelpers.remove(index)
+                              }
+                            >
+                              <IconButton
+                                color="secondary"
+                                aria-label="add to shopping cart"
+                              >
+                                <DeleteIcon fontSize="small" />
+                              </IconButton>
+                            </InputAdornment>
+                          )
+                        }}
+                      />
+                    </div>
+                  );
+                })
+              ) : (
+                <></>
+              )}
```

Comment:
I think You need use just null instead of empty fragment or <></>, cause it's not make sense.

Reply:
I think he need to use a logical operator && instead of a ternary operator.